files/jira.rb

```diff
@@ -5,10 +5,15 @@
 class Jira < BaseHandler
 
   def build_labels
-    [ "SENSU_#{@event['client']['name']}",
+    [
+      "SENSU_#{@event['client']['name']}",
       "SENSU_#{@event['check']['name']}",
-      "SENSU", *@event['check']['tags'] ].uniq.reject { |x| x.nil? }.map { |x|
-        x.strip.gsub(/\s+/, '_') }
+      "SENSU",
+      *@event['check']['tags'],
+      *team_data('tags'),
```

Comment:
Is this something you are introducing? I just checked and we don't have team tags anywhere.